charts/cluster-api-cluster-openstack/templates/control-plane.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
   labels:
     {{- include "openstackcluster.labels" . | nindent 4 }}
   annotations:
+    controlplane.cluster.x-k8s.io/skip-kube-proxy: "{{ .Values.controlPlane.skipKubeProxy }}"
```

Comment:
Just double checking the UX here, do we really need to specify an annotation _and_ then explicitly skip a phase?  Seems a bit shit, but then again this is CAPO...

Reply:
Yeah... honestly I'm not totally sure on their reasoning for this annotation. I didn't actually get around to trying without - but I meant to and will - so glad you bought that up.

But from what I can gather this prevents a reconcile (according to the 7 words of "documentation" it supplies relating to this) of kube-proxy. Now this could be a legacy thing that existed before the allowed `skipPhases` was added oOR it's some hacky work around to prevent it bringing it back in on upgrade. For context, this is the docs page which as you can see, it's as usless as a chocolate teapot. https://cluster-api.sigs.k8s.io/reference/labels_and_annotations

Let me fire a test without it and see what happens before we merge this in.